src/sys/expressionparser.cxx

```diff
@@ -149,6 +250,41 @@ FieldGeneratorPtr ExpressionParser::parseIdentifierExpr(LexInfo& lex) const {
   string name = lowercase(lex.curident);
   lex.nextToken();
 
+  // sum(symbol, count, expr)
+  // e.g. sum(i, 10, {i}^2) -> 0 + 1^2 + 2^2 + 3^2 + ... + 9^2 
+  if (name == "sum") {
+    if (lex.curtok != '(') {
+      throw ParseException("Expecting ')' after 'sum' in 'sum(symbol, count, expr)'");
```

Comment:
Should this be `"Expecting '(' after 'sum'...`?

Reply:
Thanks, good catch!